libraries/AC_Fence/AC_Fence.cpp

```diff
@@ -316,7 +316,11 @@ bool AC_Fence::pre_arm_check(const char* &fail_msg) const
 
     // check no limits are currently breached
     if (_breached_fences) {
-        fail_msg =  "vehicle outside fence";
+        //GCS_SEND_TEXT(MAV_SEVERITY_DEBUG, "### _breached_fences %d", _breached_fences);
+        static char text[24];
+        memset(text, 0, sizeof(text));
+        hal.util->snprintf(text, 24, "vehicle outside fence %1d", _breached_fences);
```

Comment:
The number is not actualy use full. Its a bit mask of fence types. I guess you could do the code to print the types, 4 is polygon fence for example. But that would cost a fair bit of flash.

Reply:
@IamPete1 san
Thanks for the comment.
I have tried to display the value of FENCE_TYPE.

![Screenshot from 2021-12-19 13-52-56](https://user-images.githubusercontent.com/646194/146664410-8d06d1f9-f9d7-4824-a0a0-803e88c70293.png)